server/services/zwave-js-ui/lib/commands/installZ2mContainer.js

```diff
@@ -0,0 +1,123 @@
+const cloneDeep = require('lodash.clonedeep');
+const { promisify } = require('util');
+const { exec } = require('../../../../utils/childProcess');
+const { EVENTS, WEBSOCKET_MESSAGE_TYPES } = require('../../../../utils/constants');
+const containerDescriptor = require('../../docker/gladys-zwavejsui-zwavejsui-container.json');
+const logger = require('../../../../utils/logger');
+const { CONFIGURATION } = require('../constants');
+
+const sleep = promisify(setTimeout);
+
+/**
+ * @description Install and starts ZwaveJSUI container.
+ * @example
+ * installZ2mContainer();
+ */
+async function installZ2mContainer() {
```

Comment:
Suggesting to rename as it can confuse with Zigbee2Mqtt

maybe 'installZwjsContainer'